sklego/preprocessing/repeatingbasis.py

```diff
@@ -116,6 +122,31 @@ def transform(self, X):
         """
         check_is_fitted(self, ["pipeline_"])
         return self.pipeline_.transform(X)
+      
+    def inverse_transform(self, X):
+        """Transform RBF features back to the input range. Outputs a numpy array. 
+
```

Comment:
Can we add what you mention in the description in the docstring Notes of the method?

In particular I am referring to:

> Note that the transformation is only invertible if the original values are in the input_range (upper bound excluded). Otherwise, the reconstructed values are only equal modulo the width of the range.